pubspec.yaml

```diff
@@ -7,7 +7,8 @@ issue_tracker: https://github.com/RevenueCat/purchases-flutter/issues
 documentation: https://docs.revenuecat.com/docs
 
 environment:
-  sdk: ">=2.12.0 <4.0.0"
+  sdk: ">=2.15.0 <4.0.0"
```

Comment:
`warning • This API is available since SDK 2.15.0, but constraints '>=2.12.0 <4.0.0' don't guarantee it • test/purchases_flutter_test.dart:810:32 • sdk_version_since`